<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="PCIEable">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PCIEable</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PCIDisable">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>PCIDisable</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>led2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>led3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>led4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>led7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>led6</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>led5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>led8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LedSet">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>LedSet</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DAC1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DAC2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DAC3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="DAC4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>34</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DAC1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>73</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DAC2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>113</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DAC3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>155</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>DAC4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="led1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>led1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DACSet">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>DACSet</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ADC1">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>30</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ADC2">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>70</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ADC3">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ADC4">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>150</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ADC5">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>190</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ADC6">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>230</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ADC7">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ADC8">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ADCGet">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>350</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ADCGet</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ADCEable">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ADCEable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="FPGA_Ver">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>150</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ADCDisable">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>ADCDisable</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="DIO">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="DIOSet">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>DIOSet</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>67</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>DIO</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>PCIEable</tabstop>
  <tabstop>led1</tabstop>
  <tabstop>led2</tabstop>
  <tabstop>led3</tabstop>
  <tabstop>led4</tabstop>
  <tabstop>led5</tabstop>
  <tabstop>led6</tabstop>
  <tabstop>led7</tabstop>
  <tabstop>led8</tabstop>
  <tabstop>LedSet</tabstop>
  <tabstop>DAC2</tabstop>
  <tabstop>DAC3</tabstop>
  <tabstop>DAC4</tabstop>
  <tabstop>DAC1</tabstop>
  <tabstop>DACSet</tabstop>
  <tabstop>ADC1</tabstop>
  <tabstop>ADC2</tabstop>
  <tabstop>ADC3</tabstop>
  <tabstop>ADC4</tabstop>
  <tabstop>ADC5</tabstop>
  <tabstop>ADC6</tabstop>
  <tabstop>ADC7</tabstop>
  <tabstop>ADC8</tabstop>
  <tabstop>ADCGet</tabstop>
  <tabstop>ADCEable</tabstop>
  <tabstop>FPGA_Ver</tabstop>
  <tabstop>ADCDisable</tabstop>
  <tabstop>PCIDisable</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <slots>
  <slot>Checked()</slot>
 </slots>
</ui>
